lucet-runtime/lucet-runtime-internals/src/module/dl.rs

```diff
@@ -186,8 +196,8 @@ impl ModuleInternal for DlModule {
         }
     }
 
-    fn trap_manifest(&self) -> &[TrapManifestRecord] {
```

Comment:
the irony is not lost on me that keeping code and trap table pointers together results in a change that's more renaming `TrapManifest` to `FunctionManifest`